requirements_dev.txt

```diff
@@ -1,15 +1,8 @@
-pytest==5.2.1
-pytest-cov==2.8.1
-coverage==4.5.4
-mock==3.0.5
-
-# PyMongo and Athena dependencies are needed for some of the unit tests:
-# (this is not perfect and we should resolve this in a different way)
-pymongo[srv,tls]==4.3.3
-boto3>=1.10.0,<1.11.0
-botocore>=1.13,<1.14.0
-PyAthena>=1.5.0,<=1.11.5
```

Comment:
During testing, we should include the dependencies of `requirements_all_ds.txt`.

Reply:
When change this file, I don't know if the grass dockerfile needs to be adjusted.

Reply:
In #6152, we have made adjustments to the dependencies required for testing.

Reply:
> In #6152, we have made adjustments to the dependencies required for testing.

Cool!